src/app/App.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import SplashScreen from "../splashScreen/SplashScreen";
+import LogInScreen from "../session/LogInScreen";
+import { HashRouter, Route } from "react-router-dom";
+import { ALBUM, INDEX, LOG_IN } from "./routes";
+import AlbumScreen from "../album/AlbumScreen";
+import "./App.css";
+
+export default class App extends Component {
+  render() {
+    return (
+      <HashRouter>
```

Comment:
The default ``BrowserRouter`` does not work with applications loaded as static files.